### PR TITLE
git-update-index: add page

### PR DESCRIPTION
**Pull request by yukuku**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/git-update-index.md [hunk 1]
```diff
@@ -3,6 +3,6 @@
 > Git command for manipulating the index.
 > More information: <https://git-scm.com/docs/git-update-index>.
 
-- Pretend that a modified file is unchanged. (`git status` will not show this as changed.)
+- Pretend that a modified file is unchanged. (`git status` will not show this as changed.):
```
Comment (zdroid) [inline pages/common/git-update-index.md]:
```suggestion
- Pretend that a modified file is unchanged (`git status` will not show this as changed):
```

**Comment by yukuku** [comment]
I've used this several times when dealing with repositories that I don't regularly maintain but has `.gitignore` that does not exclude temporary/build files that are produced on my machine. 

e.g.

1. A repository does not exclude the `out/` directory but when I build the project, the intermediate files are stored by default in `out/` directory. I'm not familiar enough with this project so I don't want to add `out/` to `.gitignore` an push that.

2. A repository has an instruction to patch certain lines in order for a hack to work, or in order to enable debug mode, or to increase network connection time out. The change modifies the source code but only for local developer's machine, it is not supposed to be commited.

File: pages/common/git-update-index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# git update-index
+
+> Git command for manipulating the index.
+> More information: <https://git-scm.com/docs/git-update-index>.
+
+- Pretend that a modified file is unchanged (`git status` will not show this as changed):
+
+`git update-index --skip-worktree {{modified_file}}`
```
Comment (einverne) [inline pages/common/git-update-index.md]:
```suggestion
`git update-index --skip-worktree {{path/to/modified_file}}`
```

Keep consistent with other pages.

**Comment by sbrl** [comment]
Wow, that's such a very cool use-case @yukuku - thanks so much! I'll remember that one - it would have made a number of situations much easier :D

**Comment by waldyrious** [comment]
Hey @yukuku and @sbrl — just thought I'd mention that for the use case of a missing `.gitignore` in a project, you can easily set a repo-specific gitignore file by editing the `.git/info/exclude` file within the repo. That's more flexible than manually updating the index. Check the [official docs](https://git-scm.com/docs/gitignore) and [this StackOverflow Q&A](https://stackoverflow.com/questions/22906851/when-would-you-use-git-info-exclude-instead-of-gitignore-to-exclude-files) for more info.

**Pull request merged**